dev/app/app.js

```diff
@@ -1,6 +1,11 @@
 'use strict';
 
 /**
+ * Change to switch between local JSON and firebase
+ */
+var localData = true;
+
```

Comment:
So every time you checkout a branch can be randomly set to `dist` or `dev` and need to check where you work from?